### PR TITLE
Specify format of the null value

### PR DESCRIPTION
**Pull request by Mats-SX**

### CODE REVIEW

**Comment by Mats-SX** [comment]
Pending CLG decision.

File: docs/style-guide.adoc [hunk 1]
```diff
@@ -195,6 +195,22 @@ RETURN "Cypher"
 RETURN 'Cypher'
 ----
 
+. Write the value null in lower case.
```
Comment (petraselmer) [inline docs/style-guide.adoc]: In its final form, this sentence is a bit hard to read... can null be highlighted somehow?

Comment (Mats-SX) [inline docs/style-guide.adoc]: How about backticks?

**Comment by petraselmer** [comment]
@Mats-SX Looks great! Just needs a rebase...

**Pull request merged**